lib/api.db.mongodb.js

```diff
@@ -21,20 +21,19 @@ if (api.mongodb) {
     // }
   ) => {
     client.connect(database.url, (err, con) => {
-      if (!err) {
-        database.connection = con;
-        api.db.mongodb.mixinDatabase(database);
-        if (database.config.collections) {
-          database.loadCollections(database.config.collections, callback);
-        } else {
-          con.collections((err, collections) => {
-            database.collections = collections.map(
-              collection => collection.collectionName
-            );
-            database.loadCollections(database.collections, callback);
-          });
-        }
-      } else callback();
+      if (err) return callback();
+      database.connection = con;
+      api.db.mongodb.mixinDatabase(database);
+      if (database.config.collections) {
+        database.loadCollections(database.config.collections, callback);
+      } else {
+        con.collections((err, collections) => {
+          database.collections = collections.map(
```

Comment:
It mean that if config contains collection list we need not load it from database.
Developer may list just needed collections.
But this solution isn't best. I think it would be better to select single way to get collection list.